creator/groups.py

```diff
@@ -100,6 +100,11 @@
         "list_all_validationrun",
         "list_all_validationresultset",
         "view_validationresultset",
+        "view_organization",
```

Comment:
`view_organization` is the 'admin' permission where anyone who has it may view any organization. The `view_my_organization` is for general users where they may only view organizations that they belong to.
This is the generally how the `view` vs `view_my` permissions work, although I may have made an exception here where even those with `view_organization` may only view their orgs because of the high-level nature of organizations. This is something to be fleshed out more in the permissions update.

Reply:
Oh oops, with the diff I didn't see these were all in Administrator group